apps/parsley/src/components/Search/SearchBar/index.tsx

```diff
@@ -220,12 +225,20 @@ const InputWrapper = styled.div`
 
 const StyledInput = styled(TextInputWithGlyph)`
   /* overwrite lg borders https://jira.mongodb.org/browse/PD-1995 */
-  div input {
+  div > div {
     border-top-left-radius: 0;
     border-bottom-left-radius: 0;
-    border-left: 0;
+    border-left: none;
+  }
+  input[data-cy="searchbar-input"] {
     padding-left: 42px;
   }
+  div[data-lgid="lg-form_field-feedback"] {
+    display: none;
+  }
```

Comment:
I'm not a huge fan of depending on internal attributes of 3rd party components like this since we can't control what they will be which is partly why this broke in the first place. Have you considered the alternative approach of not using this component's "error" state? We are overriding the error state anyways so there doesn't seem to be much value in using it.

Reply:
I opted to utilize the error state because it comes with the border-color. When it comes to hacking on packages, I rather take things away than add things in to follow the the original pattern as closely as possible.